client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -0,0 +1,628 @@
+	const overallSpendingFormatted = `$${ formatCents( overall_spending || 0 ) }`;
+	const deliveryEstimateFormatted = getCampaignEstimatedImpressions( display_delivery_estimate );
+	const campaignTitleFormatted = title || __( 'Untitled' );
+	const campaignCreatedFormatted = moment.utc( start_date ).format( 'MMMM DD, YYYY' );
+	const devicesListFormatted = devicesList ? `${ devicesList }` : __( 'All' );
+	const countriesListFormatted = countriesList ? `${ countriesList }` : __( 'All' );
+	const osListFormatted = OSsList ? `${ OSsList }` : translate( 'All' );
+	const topicsListFormatted = topicsList ? `${ topicsList }` : __( 'All' );
+	const impressionsTotal = formatNumber( impressions_total );
+	const subtotalFormatted = `$${ formatCents( subtotal || 0 ) }`;
+	const creditsFormatted = `$${ formatCents( credits || 0 ) }`;
+	const totalFormatted = `$${ formatCents( total || 0 ) }`;
+
+	const navigationItems = isSmallScreen
+		? [
+				{
+					label: (
+						<>
+							<Gridicon icon="chevron-left" size={ 18 } />
+							{ translate( 'Back' ) }
+						</>
+					),
+					href: `/advertising`,
+				},
+		  ]
+		: [
+				{ label: translate( 'Advertising' ), href: `/advertising` },
+				{ label: campaignTitleFormatted || '', href: `/campaigns/${ campaignId }` },
+		  ];
+
+	const icon = (
+		<span className="campaign-item-details__support-buttons-icon">
+			<svg
+				width="16"
+				height="17"
+				viewBox="0 0 16 17"
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+			>
+				<path
+					fillRule="evenodd"
+					clipRule="evenodd"
+					d="M8 16.3193C12.4183 16.3193 16 12.7376 16 8.31934C16 3.90106 12.4183 0.319336 8 0.319336C3.58172 0.319336 0 3.90106 0 8.31934C0 12.7376 3.58172 16.3193 8 16.3193ZM13.375 11.9755C14.085 10.9338 14.5 9.67502 14.5 8.31934C14.5 7.08407 14.1554 5.92928 13.5571 4.94585L12.2953 5.75845C12.7428 6.50748 13 7.38337 13 8.31934C13 9.37572 12.6724 10.3556 12.1132 11.1629L13.375 11.9755ZM11.4245 13.8451L10.6121 12.5836C9.85194 13.0503 8.95739 13.3193 8 13.3193C7.04263 13.3193 6.1481 13.0503 5.38791 12.5836L4.57552 13.8451C5.56993 14.4627 6.74332 14.8193 8 14.8193C9.2567 14.8193 10.4301 14.4627 11.4245 13.8451ZM2.62498 11.9755C1.91504 10.9338 1.5 9.67503 1.5 8.31934C1.5 7.08405 1.84458 5.92926 2.44287 4.94582L3.70473 5.75842C3.25718 6.50745 3 7.38336 3 8.31934C3 9.37573 3.32761 10.3556 3.88678 11.1629L2.62498 11.9755ZM5.20588 4.17229C6.00361 3.63376 6.96508 3.31934 8 3.31934C9.03494 3.31934 9.99643 3.63377 10.7942 4.17232L11.6065 2.91084C10.5746 2.22134 9.33424 1.81934 8 1.81934C6.66578 1.81934 5.42544 2.22133 4.39351 2.91081L5.20588 4.17229ZM8 11.8193C9.933 11.8193 11.5 10.2523 11.5 8.31934C11.5 6.38634 9.933 4.81934 8 4.81934C6.067 4.81934 4.5 6.38634 4.5 8.31934C4.5 10.2523 6.067 11.8193 8 11.8193Z"
+					fill="#1E1E1E"
+				/>
+			</svg>
+		</span>
+	);
+
+	const databars = [
+		{
+			label: translate( 'Ad' ),
+			value: clicks_total || 0,
```

Comment:
We use these bars to show impressions:
<img width="629" alt="image" src="https://github.com/Automattic/wp-calypso/assets/115007291/00df4f8e-83a1-4316-9a94-87cee896a928">

But there we show clicks. Should be use `impressions_total || 0` instead?

Reply:
That makes "sense" in a way:
You are comparing organic views (the ones that come from stats) VS ad views. 
The ad views are actually clicks. So 1 click in the ad = 1 impression

Reply:
Got it, I was wrong.